### ISSUE TITLE
Two CAMx readers

### ISSUE THREAD

**Issue by zmoon** [noaa-oar-arl/monetio#206]
Maybe we can combine them into one that lives at `models/camx.py` for v0.3
* https://github.com/noaa-oar-arl/monetio/blob/develop/monetio/models/camx.py (old one)
* https://github.com/noaa-oar-arl/monetio/blob/develop/monetio/models/_camx_mm.py (new one)

Sorry I spaced on this in #185 / #187 @blychs .

**Comment by blychs** [comment]
If we don't need to stick to the `_model_mm.py` naming convention for melodies monet, I am happy to do that.

**Comment by zmoon** [comment]
Yes for monetio v0.3 we plan to move away from that convention. Hopefully with tests (some `_mm.py` ones already have).
